uaclient/apt.py

```diff
@@ -58,7 +58,7 @@ def assert_valid_apt_credentials(repo_url, username, password):
     except util.ProcessExecutionError as e:
         if e.exit_code == 100:
             stderr = str(e.stderr).lower()
-            if re.search(r"401\s+unauthorized|httperror401", stderr):
+            if re.search(r"401\s+[uU]nauthorized|httperror401", stderr):
```

Comment:
I wouldn't expect this to matter because we call `.lower()` on the output on the line above.

Did this change fix something?

Reply:
Huh! I did not see the .lower() call, my bad!